static/app/views/issueList/overview.tsx

```diff
@@ -1073,7 +1080,16 @@ class IssueListOverview extends React.Component<Props, State> {
     }
   };
 
-  onIssueAction = (itemIds: string[]) => {
+  onIssueAction = (itemIds: string[], actionType: string) => {
+    if (itemIds.length > 1) {
+      addMessage(t(`${actionType} ${itemIds.length} Issues`), 'success', {
+        duration: 4000,
+      });
+    } else {
+      const shortId = itemIds.map(item => GroupStore.get(item)?.shortId).toString();
+      addMessage(t(`${actionType} ${shortId}`), 'success', {duration: 4000});
```

Comment:
Why do we set a custom duration?

Reply:
I set this to be the same as `showAlert` that the original delete/merge used
https://github.com/getsentry/sentry/blob/6b4dc91d5cf5d886a15cca7e05dc91d8613f2707/static/app/stores/groupStore.tsx#L17-L19

I'll also be adding an "Undo" button to the toast (except for delete/merge), and wanted to duration to be the same for all of them